### PR TITLE
30 - Pause screen with "Resume Game" and "Back to login" buttons. 

### PR DESCRIPTION
**Pull request by Jvstinoo**
Added button to pause game which takes you to a pause screen with "Resume Game" and "Back to login" buttons.

Screenshot 2024-09-17 001733
Screenshot 2024-09-17 001741

### CODE REVIEW

File: src/App.js
Comment (G4vp) [inline src/App.js:1]: Es este archivo necesario? Ya estas llamando <PauseScreen> en frontend/src/App.js

**Review by fernandopizarro1 — CHANGES_REQUESTED** [review]
I dont really think the `App.js` file is really necessary, since the frontend is the one that runs the game.

File: src/App.js
Comment (Jvstinoo) [reply src/App.js:1]: Correcto, es un archivo viejo ya que habia clonado el repo antes de que se crearan los otros folders. Removido.

**Comment by Jvstinoo** [comment]
> I dont really think the `scr/App.js` file is really necessary, since the game runs from the frontend folder.

That's right, it was an old file I forgot to remove since I cloned the repo a few weeks back when the other folders weren't created. Removed.

**Review by fernandopizarro1 — APPROVED** [review]
Nice job! LGTM

**Review by AngelCIICMorales — CHANGES_REQUESTED** [review]
Whenever you pause the game it resets the game.

**Comment by Jvstinoo** [comment]
> Whenever you pause the game it resets the game.

Wasn't aware that was part of this issue. Added the functionality to save game while paused.

**Review by AngelCIICMorales — APPROVED** [review]
LGTM!

**Pull request merged**